packages/fx-core/src/component/generator/spfxGenerator.ts

```diff
@@ -167,16 +172,21 @@ export class SPFxGenerator {
       if (solutionName) {
         args.push("--solution-name", solutionName);
       }
-      await cpUtils.executeCommand(
-        destinationPath,
-        context.logProvider,
-        {
-          timeout: 2 * 60 * 1000,
-          env: yoEnv,
-        },
-        "yo",
-        ...args
-      );
+
+      try {
+        await cpUtils.executeCommand(
+          destinationPath,
+          context.logProvider,
+          {
+            timeout: 2 * 60 * 1000,
+            env: yoEnv,
+          },
+          "yo",
+          ...args
+        );
+      } catch (yoError) {
```

Comment:
If we throw YoGeneratorScaffoldError directly here, we'll have difficulties troubleshooting scaffolding error. Better throw the original error message as well.

Reply:
synced offline and we will put such details in output window,